spec/models/project_spec.rb

```diff
@@ -1,6 +1,41 @@
 require File.dirname(__FILE__) + '/../spec_helper.rb'
 
 describe Project do
+  xit { is_expected.to belong_to :user }
+  xit { is_expected.to belong_to(:place).inverse_of :projects }
```

Comment:
With the 5.x switch to requiring `belongs_to` by default and then globally setting it to false, I've marked a few of these pending. I think the best solution would be to rewrite the association:

```ruby
belongs_to :user, optional: false, inverse_of: :projects
```
and remove the separate presence validation, but I'll leave those changes to another PR (hopefully after factories merges).